Content.Shared/Follower/FollowerSystem.cs

```diff
@@ -102,7 +102,7 @@ private void OnGetAlternativeVerbs(GetVerbsEvent<AlternativeVerb> ev)
         {
             var verb = new AlternativeVerb()
             {
-                Priority = 10,
+                Priority = -10, // Starlight edit: Prioritize actions over follow for Aghost
```

Comment:
make it so priority changes if you're aghost:

```suggestion
                Priority = ghost.CanGhostInteract ? -10 : 10, // Starlight edit: Prioritize actions over follow for Aghost
```

(if you're accept this change, also needs to chance HasComp to TryComp)